watch.d/ABOS_SOTS.json

```diff
@@ -1,5 +1,5 @@
 {
   "path": [ "ABOS/SOTS" ],
   "execute": "ABOS/common/incoming_handler.sh",
```

Comment:
I guess the handler you just removed
``` ABOS/SOTS/incoming_handler.sh``` hasn't be used for a long time anyway right ?